src/App.js

```diff
@@ -10,6 +10,7 @@ import StudentServicesPage from "./pages/StudentServices";
 import CSSAPage from "./pages/CSSA";
 import AboutUsPage from "./pages/AboutUs";
 import MainNavigation from './components/layout/MainNavigation';
+import GermanWebsite from './pages/DE/GermanWebsite'
```

Comment:
It seems we use double quotes for most cases. We may also want to fix that for imports above and below this line.  
```suggestion
import GermanWebsite from "./pages/DE/GermanWebsite";
```